### DISCUSSION TITLE
dodog cli not available

### DISCUSSION THREAD

**Discussion by anishnagaraj** [cucumber/godog#503] [General]
Hi Guys!

I want to use godog cli to create the step definition of my feature file. I used the following to install the command  

`go install github.com/cucumber/godog/cmd/godog@latest`

But I am unable to know where the tool is getting installed and so it not set to the PATH variable. Hence I get `zsh: command not found: godog`

Has anybody faced this problem? Btw, I am using a macbook

**Comment by vearutop** [comment]
The `go install` command installs binaries to its own directory, by default `godog` would be available as `$(go env GOPATH)/bin/godog`.

It is common to add that directory to your `PATH`, so that installed tools are easily available.

Please check https://stackoverflow.com/a/70098877/329463.

**Reply by anishnagaraj** [reply]
I am a newbie to goland and this input is really helpful! thank you @vearutop. This worked!
